executor/admin.go

```diff
@@ -97,6 +97,9 @@ func (e *CheckIndexRangeExec) Open(ctx context.Context) error {
 		ID:   model.ExtraHandleID,
 		Name: model.ExtraHandleName,
 	})
+	for i := range e.cols {
+		e.cols[i].FieldType = *e.schema.Columns[i].RetType
```

Comment:
Question: this assign to `e.cols[i].FieldType` whether have risk to modify the shared variable `infoschema.infoSchemaDB` ?_?

Reply:
nice catch, we should only fix the field type of the newly added column.